app/src/main/java/org/shadowice/flocke/andotp/Activities/SettingsActivity.java

```diff
@@ -148,7 +148,7 @@ public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
         if (fragment.useAutoBackup != null) {
             fragment.useAutoBackup.setEnabled(BackupHelper.autoBackupType(this) == Constants.BackupType.ENCRYPTED);
             if (!fragment.useAutoBackup.isEnabled())
-                fragment.useAutoBackup.setChecked(false);
+                fragment.useAutoBackup.setValue(Constants.AutoBackup.OFF.toString());
```

Comment:
Needs ``.toLowerCase(Locale.ENGLISH)`` if we change the resource values to lower case.

Reply:
Do you mean to upper?

Reply:
Oh no, I see what you mean